### PR TITLE
feat(agora): migration News component (AG-1546)

### PR DESCRIPTION
**Pull request by sagely1**
## Description

Migration of News Component

## Related Issue

https://sagebionetworks.jira.com/browse/AG-1546

## Changelog

- Add news component
   
## Testing
`nx test agora-news`

## Expected output
Navigate to:
http://localhost:4200/news

image

### CODE REVIEW

File: libs/agora/teams/src/lib/teams/teams.component.spec.ts [hunk 1]
```diff
@@ -6,8 +6,8 @@ describe('TeamListComponent', () => {
   let component: TeamsComponent;
   let fixture: ComponentFixture<TeamsComponent>;
 
-  beforeEach(async () => {
-    await TestBed.configureTestingModule({
+  beforeEach(() => {
```
Comment (sagely1) [inline libs/agora/teams/src/lib/teams/teams.component.spec.ts:9]: not using any async here so doing some code cleanup

File: libs/agora/teams/src/index.ts [hunk 2]
```diff
@@ -1,2 +1,3 @@
 export * from './lib/teams/teams.routes';
 export * from './lib/team-list/team-list.routes';
+export * from './lib/team-member-list/team-member-list.routes';
```
Comment (sagely1) [inline libs/agora/teams/src/index.ts:3]: adding missing route

File: libs/agora/testing/src/lib/server/index.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export * from './msw-server';
```
Comment (sagely1) [inline libs/agora/testing/src/lib/server/index.ts:1]: added the /server folder to make it clearer on imports which 'server' is being exported

File: tsconfig.base.json [hunk 4]
```diff
@@ -21,10 +21,12 @@
       "@sagebionetworks/agora/api-client-angular": ["libs/agora/api-client-angular/src/index.ts"],
       "@sagebionetworks/agora/config": ["libs/agora/config/src/index.ts"],
       "@sagebionetworks/agora/models": ["libs/agora/models/index.ts"],
+      "@sagebionetworks/agora/news": ["libs/agora/news/src/index.ts"],
       "@sagebionetworks/agora/not-found": ["libs/agora/not-found/src/index.ts"],
       "@sagebionetworks/agora/services": ["libs/agora/services/src/index.ts"],
       "@sagebionetworks/agora/teams": ["libs/agora/teams/src/index.ts"],
-      "@sagebionetworks/agora/testing/mocks": ["libs/agora/testing/src/lib/mocks"],
+      "@sagebionetworks/agora/testing/mocks": ["libs/agora/testing/src/lib/mocks/index.ts"],
+      "@sagebionetworks/agora/testing/server": ["libs/agora/testing/src/lib/server/index.ts"],
```
Comment (tschaffter) [inline tsconfig.base.json:29]: The existing convention is to have one item listed in this file per library (project). Are those two items needed?

Comment (sagely1) [reply tsconfig.base.json:29]: I'll change to a single item.

Comment (tschaffter) [reply tsconfig.base.json]: Thanks!

**Review by tschaffter — APPROVED** [review]

**Pull request merged**